app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,14 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'do_not_answer@dani-and-manu.info'
   layout 'mailer'
+
+  def new_request(request_id)
+
+    @request = Request.find_by(id: request_id)
+    @user = User.find_by(id: @request.user_id)
```

Comment:
This will do the exact same thing, thanks to Active Record relations :
```ruby
@user = @request.user
```

Because in our model `request.rb` we have the following:
```ruby
# app/models/request.rb

class Request < ApplicationRecord
# [...]
  belongs_to :user
# [...]
end
```